docs/source/whatsnew/version3.rst

```diff
@@ -66,6 +66,12 @@ characters. Python 3 allows unicode identifiers, and IPython 3 makes it easier
 to type those, using a feature from Julia. Type a backslash followed by a LaTeX
 style short name, such as ``\alpha``. Press tab, and it will turn into α.
 
+Widget migration guide
+----------------------
+The widget framework has a lot of backwards incompatible changes.  For information about migrating widget notebooks and custom widgets to 3.0 refere to the `widget migration guide`_.
+
+.. _widget migration guide: ./version3_widget_migration.rst
```

Comment:
@takluyver @minrk I have no idea if this link format works for local files too.  Let me know if this is incorrect.